JellyfinPlayer/Components/PortraitItemView.swift

```diff
@@ -47,7 +47,7 @@ struct PortraitItemView: View {
                                 Image(systemName: "checkmark.circle.fill")
                                     .foregroundColor(.accentColor)
                                     .background(Color(.white))
```

Comment:
If we're thinking in the mindset of setting background/foreground colors directly on the icon, this icon supports multicolor mode from iOS 14 & up (i.e. the intended targets)
<img width="266" alt="Screen Shot 2021-07-31 at 7 50 24 AM" src="https://user-images.githubusercontent.com/34462061/127738994-f7992539-1a6c-4066-a6e4-db7adafd5e94.png">

Reply:
I'd rather do this then continue with my hacky way of doing this :p

Reply:
The multicolor support in iOS 14 **only** allows for it to be that blue color with the white checkmark.

In iOS 15, we can change the palette of the colors for the symbol but it is only supported in iOS 15.

Look at section: _How to create a custom SF Symbols palette_
https://www.hackingwithswift.com/articles/237/complete-guide-to-sf-symbols

Reply:
The hack you did was good and it's necessary here

Reply:
> The multicolor support in iOS 14 _only_ allows for it to be that blue color with the white checkmark.
> 
> In iOS 15, we can change the palette of the colors for the symbol but it is only supported in iOS 15.
> 
> Look at section: _How to create a custom SF Symbols palette_
> https://www.hackingwithswift.com/articles/237/complete-guide-to-sf-symbols

Oh, didn't know that. That wouldn't work as we use the purple accent color.
this lgtm!